jobs/cjd_sample/src/main/java/com/google/samples/BatchOperationSample.java

```diff
@@ -0,0 +1,148 @@
+package com.google.samples;
```

Comment:
Needs license header. Ditto for the other files. See http://go/releasing/preparing#headers

Reply:
Done.